attribute-masking-service/README.md

```diff
@@ -35,37 +39,37 @@ The service performs two functions:
 | user                    | maskedLeafResource       | exception         | user
 | leafResource            |                          | serverMetadata    | leafResource
 | rules                   |                          |                   | rules
-(fields marked with * are acquired from headers metadata)
 
-The service takes in an `AttributeMaskingRequest` and a request `Token`.
-This request is persisted in a store as an `AuthorisedRequestEntity`.
-The `LeafResource` attached to the request is then masked, removing any excessive or sensitive metadata which a client may be prohibited from accessing.
-The resulting masked `LeafResource` is then attached to an `AttributeMaskingResponse` and outputted from the service.
-
-### Future Enhancements
-The masking operation may in the future apply attribute-level `Rule`s using the `User` and `Context` for fine-grained control over client metadata access.
+*token is in the message header's metadata
 
+The service reads a message in from the Kafka "rule" topic as a [AttributeMaskingRequest](src/main/java/uk/gov/gchq/palisade/service/attributemask/model/AttributeMaskingRequest.java).
```

Comment:
```suggestion
The service reads a message in from the Kafka `rule` topic as an [AttributeMaskingRequest](src/main/java/uk/gov/gchq/palisade/service/attributemask/model/AttributeMaskingRequest.java).
```

Reply:
The "an" instead of "a", definitely and have changed.  With the quotes surrounding the word rule, I have kept the double quotes.  The intent is to show that the word rule has a special meaning.  In this case it is the type of topic and is used in the same way as "masked-resource" and "error".

Reply:
Reviewed other examples in the code and it looks like I am wrong.  we are using a convention with the special single  quote is being used in other readme's.  I'll change this to be inline with this convention.